docs/configuration.md

```diff
@@ -282,6 +331,12 @@ It is possible to specify exactly which code blocks to mutate by means of a _mut
 
 _Note:_ It is not possible to combine mutation range with a globbing expression in the same line.
 
+**Other examples:**
+If you want to only mutate one specific file:
+
+- `--mutate src/app/home/home.component.ts`, or if you want to mutate just one directory:
+- `--mutate "src/app/home/*.ts","!src/app/home/*.spec.ts"`
+
```

Comment:
yes, moved the examples to the "config file" and "command line" examples a few lines above so that all examples are at one place,  and removee duplication